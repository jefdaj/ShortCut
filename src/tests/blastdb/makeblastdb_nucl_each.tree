.
|-- cache
|   |-- lines
|   |   |-- 0815de9417.txt
|   |   |-- 1f98704802.txt
|   |   |-- 27a269e49d.txt -> ../../cache/lines/e1e5fdbeba.txt
|   |   |-- 3ce4648d84.txt
|   |   |-- 615fc32080.txt
|   |   |-- b1e5ca661f.txt -> ../../cache/lines/d860635d17.txt
|   |   |-- c9fce61422.txt
|   |   |-- cbf12a91ca.txt
|   |   |-- d860635d17.txt
|   |   |-- e1e5fdbeba.txt
|   |   `-- f058e7d872.txt
|   |-- load
|   |   |-- 6ea57f1208.fna -> $WORKDIR/data/6803genes10.fna
|   |   `-- 737dfdb34e.fna -> $WORKDIR/data/6803genes.fna
|   |-- makeblastdb
|   |   |-- d860635d17
|   |   |   |-- d860635d17.ndb.nhr
|   |   |   |-- d860635d17.ndb.nin
|   |   |   `-- d860635d17.ndb.nsq
|   |   `-- e1e5fdbeba
|   |       |-- e1e5fdbeba.ndb.nhr
|   |       |-- e1e5fdbeba.ndb.nin
|   |       `-- e1e5fdbeba.ndb.nsq
|   `-- map
|       |-- 52c6bf3aa8.ndb -> ../../cache/lines/615fc32080.txt
|       `-- 53fe259f1b.ndb -> ../../cache/lines/c9fce61422.txt
|-- exprs
|   |-- list
|   |   `-- f283a14ffc.fna.list -> ../../cache/lines/3ce4648d84.txt
|   |-- load_fna
|   |   |-- 9b7934c579.fna -> ../../cache/load/6ea57f1208.fna
|   |   `-- f110b4b1c9.fna -> ../../cache/load/737dfdb34e.fna
|   |-- makeblastdb_nucl_each
|   |   `-- 9087a5152d.ndb.list -> ../../cache/lines/cbf12a91ca.txt
|   |-- singletons
|   |   `-- bb5d85d845.fna.list.list -> ../../cache/lines/f058e7d872.txt
|   `-- str
|       |-- 7da80bde02.str -> ../../cache/lines/0815de9417.txt
|       `-- a2fd369878.str -> ../../cache/lines/1f98704802.txt
`-- vars
    |-- dbs.ndb.list -> ../exprs/makeblastdb_nucl_each/9087a5152d.ndb.list
    |-- large6803fna.fna -> ../exprs/load_fna/f110b4b1c9.fna
    |-- result -> ../vars/dbs.ndb.list
    `-- small6803fna.fna -> ../exprs/load_fna/9b7934c579.fna

14 directories, 32 files
