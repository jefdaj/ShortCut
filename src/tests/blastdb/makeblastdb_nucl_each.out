[BLAST nucleotide database $TMPDIR/cache/map/53fe259f1b.ndb,
 BLAST nucleotide database $TMPDIR/cache/map/52c6bf3aa8.ndb]
